rs/sns/swap/src/swap.rs

```diff
@@ -881,7 +881,8 @@ impl Swap {
 
             for cf_neuron in cf_participant.cf_neurons.iter_mut() {
                 // Create a closure to ensure `global_cf_memo` is incremented in all cases
-                let mut process_cf_neuron = || {
+                let hotkeys = cf_neuron.hotkeys.clone().unwrap_or_default();
```

Comment:
Would deconstructing the `cf_neuron` neuron instance allow avoiding `clone()` in this case?

Reply:
It seems to be not very easy to remove the .clone()